recipes/cvxpy/meta.yaml

```diff
@@ -0,0 +1,99 @@
+package:
+  name: cvxpy
+  version: "0.4.2"
+
+source:
+  git_url: https://github.com/cvxgrp/cvxpy
+#  patches:
+   # List any patch files here
+   # - fix.patch
+
+# build:
+  # noarch_python: True
+  # preserve_egg_dir: True
+  # entry_points:
+    # Put any entry points (scripts to be generated automatically) here. The
+    # syntax is module:function.  For example
+    #
+    # - cvxpy = cvxpy:main
+    #
+    # Would create an entry point called cvxpy that calls cvxpy.main()
+
+
+  # If this is a new build for the same version, increment the build
+  # number. If you do not include this key, it defaults to 0.
+  # number: 1
+build:
+  noarch_python: True
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+
+requirements:
+  build:
+    - setuptools
+    - six
+    - toolz
+    - cvxcanon >=0.0.22
+    - fastcache
+    - python
+    - ecos >=2
+    - scs >=1.1.3
+    - multiprocess
+    - numpy >=1.9
+    - scipy >=0.15
```

Comment:
I'm guessing we don't need all of the as build dependencies. Normally `python` and possibly `setuptools` (if used) are sufficient. Unless of course these are being imported by `setup.py`.